src/profiler/debugger.cpp

```diff
@@ -52,7 +52,7 @@ void Debugger::notifyNewThread(DWORD threadId, HANDLE threadHandle)
 bool Debugger::attach(std::function<void(NotifyData const &notification)> notifyFunc_)
 {
 	assert(!processHandle);
-	assert(!debuggerThread);
+	//assert(!debuggerThread);

```

Comment:
If there is a reason to disable the assert, please just remove it. Few things cause anxiety-filled uncertainty when trying to understand a codebase like a commented-out assert :)

Reply:
The assert won't compile, and I wasn't certain what it was referring to. (What is `debuggerThread`?) Out of caution, I chose not to remove it, thinking someone else could make a better decision.

Reply:
I see, it's a leftover of #88, but nobody tried to build Very Sleepy in the debug profile, so this wrongful addition went unnoticed. In this case feel free to add a commit which removes such left-overs.